docs/examples/numba_func/Numba_func.ipynb

```diff
@@ -0,0 +1,370 @@
+{
```

Comment:
<p>same here, the line is a bit too long. I'd break the pipeline instantiation into about 3 lines or so</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/2886/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>